test/testsimplifyusing.cpp

```diff
@@ -1588,7 +1584,7 @@ class TestSimplifyUsing : public TestFixture {
                             "STAMP(B, A);\n"
                             "STAMP(C, B);\n";
         (void)tok(code, dinit(TokOptions, $.preprocess = true));
-        ASSERT(startsWith(errout_str(), "[test.cpp:6]: (debug) Failed to parse 'using C = S < S < S < int"));
+        TODO_ASSERT(startsWith(errout_str(), "[test.cpp:6]: (debug) Failed to parse 'using C = S < S < S < int"));
```

Comment:
@chrchr-github This was producing an error because the code contained the preprocessed and non-preprocessed source because it was tokenized twice (I tracked it back to the introduction and it has always been that way). I think this is currently fine because it was about a hang/crash but if we expect to test for the error we might need to add another test.

Reply:
The original code was added in #5018.

Reply:
What is `errout_str()` after the change? I agree the output was not important, since the issue was a hang.

Reply:
It is empty.

Reply:
That even seems like progress. But then I don't understand why the refactoring made the debug message go away.

Reply:
> But then I don't understand why the refactoring made the debug message go away.

See previous comment.

> > This was producing an error because the code contained the preprocessed and non-preprocessed source because it was tokenized twice

I was asking because I think we should have a test which actually detected the debug message.